metadata/ota_metadata/metadata_gen.py

```diff
@@ -191,15 +191,63 @@ def gen_metadata(
     target_abs = Path(os.path.abspath(target_dir))
     ignore = ignore_rules(target_dir, ignore_file)
 
+    # If ignore file has the following directories:
+    #    1, "home/autoware/*/build"
+    #    2, "home/autoware/*/src"
```

Comment:
based on the pattern, it will be:
```suggestion
    #    1, "home/autoware/**/build"
    #    2, "home/autoware/**/src"
```